src/main/java/com/google/javascript/gents/GentsCodeGenerator.java

```diff
@@ -32,8 +32,10 @@ protected void add(Node n, Context ctx) {
 
     String comment = nodeComments.getComment(n);
     if (comment != null) {
-      add(comment);
-      add("\n");
+      // CodeGernator.add("\n") doesn't append anything. Fixing the actual bug in Closure Compiler
+      // is difficult. Works around the bug by passing " \n".
```

Comment:
Add that the spacebar doesn't make it into the final emit, because it is stripped by Closure.